test/mocks/mocks.go

```diff
@@ -380,7 +382,9 @@ func ServiceBindingRequestMock(
 	applicationResourceRef string,
 	matchLabels map[string]string,
 	bindUnannotated bool,
+	groupVersionResource string,
```

Comment:
Please use [schema.GroupVersionResource](https://godoc.org/k8s.io/apimachinery/pkg/runtime/schema#GroupVersionResource) here.

Reply:
Oh, thanks for this I didn't know!